syntax = "proto3";

package edu.uci.ics.amber.engine.architecture.sendsemantics;
import "edu/uci/ics/amber/engine/common/virtualidentity.proto";

message DataSendingPolicy{
  edu.uci.ics.amber.engine.common.LinkIdentity policyTag = 1;
  int32 batchSize = 2;
  repeated edu.uci.ics.amber.engine.common.ActorVirtualIdentity receivers = 3;
}


